Automattic/wp-calypso
# Stats: Refactor main chart tabs styles

#### Proposed Changes

* Introduce modernized chart tabs styles as a shared file.
* Replace the className `chart-tabs--new-main-chart` with `stats--new-main-chart` for gating new features.
* Adjust new main chart tabs styles within the `.stats--new-main-chart` element.

#### Testing Instructions



* Refer to the [Traffic page testing instructions](https://github.com/Automattic/wp-calypso/pull/69529).
* Refer to the [Store page testing instructions](https://github.com/Automattic/wp-calypso/pull/69661).
* Refer to the [Ads page testing instructions](https://github.com/Automattic/wp-calypso/pull/69743).
* Ensure the styles of the main chart work as previously.

<img width="1123" alt="截圖 2022-11-03 下午10 59 38" src="https://user-images.githubusercontent.com/6869813/199756649-e17a2b23-d578-463e-b8ad-2f843b65fd19.png">

<img width="1091" alt="截圖 2022-11-03 下午11 00 19" src="https://user-images.githubusercontent.com/6869813/199756685-825f0ae4-20b3-4a97-853c-6a37a250879f.png">

<img width="1117" alt="截圖 2022-11-03 下午10 26 02" src="https://user-images.githubusercontent.com/6869813/199747212-cdfddc1c-392d-440f-8ddc-ab8edca3ba7e.png">

#### Pre-merge Checklist






Related to #69041
